plugins/catalog/src/js/repositories/RepositoriesRouting.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { Route } from "react-router";
+import { decorate, injectable } from "inversify";
+import RepositoriesTab from "./RepositoriesTab";
+
+class RepositoriesRouting {
+  getRoutes(_partialNextState) {
```

Comment:
Why do we need the parameter if we are not using it?

Reply:
This is a reference implementation, so I left those args to demonstrate the interface.

Reply:
Gotcha